esm/worker/class.js

```diff
@@ -49,6 +49,10 @@ export default (...args) =>
             sync: {
                 value: coincident(worker, JSON).proxy,
             },
+            onerror: {
```

Comment:
OK, this is not configurable nor writable so it breaks possible extend on error. I will clarify how the error dance works but in general we should be good by adding a default `console.error` as `worker.onerror` where the worker is defined in the `_template.js`.

Reply:
:facepalm: ... the `worker` is not the one on the main thread, so your initial patch was actually good, I got mislead but the import but that import returns a real `Worker`, not the one exposed in the worker itself.

I suggest we simply do this instead:

```js
            onerror: {
                writable: true,
                configurable: true,
                value: console.error
            }
```

so that it's still possible to re-configure it by users, but by default we at least have an error shown in console.

Big apologies I should've tried harder with your code instead and look closer.